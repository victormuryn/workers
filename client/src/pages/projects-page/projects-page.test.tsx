import React from 'react';
import {BrowserRouter as Router} from 'react-router-dom';
import renderer from 'react-test-renderer';
import ProjectsPage from './projects-page';

describe(`ProjectsPage tests`, () => {
  it(`ProjectsPage renders corrects`, () => {
    const tree = renderer.create(<Router><ProjectsPage /></Router>).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
