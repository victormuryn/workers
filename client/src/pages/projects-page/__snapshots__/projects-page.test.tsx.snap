// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProjectsPage tests ProjectsPage renders corrects 1`] = `
Array [
  <h1
    className="text-center mt-5 pt-3"
  >
    Усі проєкти
  </h1>,
  <ul
    className="projects__list container"
  >
    <div
      className="row"
    >
      <div
        className="col-md-2"
      >
        Settings
      </div>
      <div
        className="col-md-10"
      >
        <h2
          className="text-center"
        >
          Не вдалося знайти жодного проєкту
        </h2>
      </div>
    </div>
  </ul>,
]
`;
